controllers/inventoriesControllers.js

```diff
@@ -84,6 +84,19 @@ const createNewInventory = async (req, res) => {
   }
 };
 
+const getAllInventoryItems = async (req, res) => {
+  try {
+    const allInventoryItems = await knex("inventories");
```

Comment:
Nit: we could consider adding these string such as "inventories" to a constants file, that way we don't have to write it every time, and if we need to refactor, we can simply do it from that file.